Libraries/ReactNative/RootViewLayoutContext.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+const React = require('React');
+
+export type LayoutContext = $ReadOnly<{|
+  frame: $ReadOnly<{|
```

Comment:
Cloud this be "layout" instead of "frame" and use the type "Layout" from "Libraries/Types/CoreEventTypes.js" instead of defining its own?

Reply:
Good point, I'll update it.